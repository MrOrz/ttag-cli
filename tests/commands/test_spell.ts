import * as path from "path";
import { execSync } from "child_process";

const poPath = path.resolve(__dirname, "../fixtures/spellTest/spell.po");

test("find basic spelling error with locale autodetect", () => {
    const result = execSync(`ts-node src/index.ts spell ${poPath}`).toString();
    expect(result).toMatchSnapshot();
});
