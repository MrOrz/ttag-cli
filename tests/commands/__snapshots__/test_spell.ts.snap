// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`find basic spelling error with locale autodetect 1`] = `
"[33mmsgid[39m \\"test_incorrect_spelling\\"
[33mmsgstr[39m \\"[4m[41mtll[49m[24m\\"


"
`;
